command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I suspect @wking's point is that while the spec doesn't require a cli (which could be just a shim to aid compliance testing) without some sort of cli or shim there would be no way - until we specify something else like a web interface spec - to validate that a particular implementation complies with the spec. Basically you can have a fully compliant Java compiler that only accepts class files via an http interface but unless someone writes a shim with the API of "javac" no compliance testing tool can validate the compliance claims.

In other words I think nothing requires you to implement (or provide a shim that implements) the cli spec to be compliant BUT if you wish to be *certified* as compliant by the compliance testing tools you probably need to implement the cli.

I think @duglin is right that it's valuable to be able to consume only a subset of a body of specs, and we should allow that. The only requirement here is that a compliance tool MAY refuse to certify a runtime that does not provide a compliant cli. Unless I'm missing something that's just an unavoidable fact.